javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,55 @@
+//SANDBOX URL: https://codesandbox.io/s/talk-to-your-computer-ph2ql?file=/src/index.js
+
+function getReply(question) {
+  switch (question) {
+    case 'Hello my name is Deepti':
+      return 'Nice to meet you Deepti';
+    case 'What is my name?':
+      return 'Your name is Deepti';
+    case 'Add fishing to my todo':
+      return 'Fishing added to your todo';
+    case 'Add singing in the shower to my todo':
+      return 'Singing in the shower added to your todo';
+    case 'Remove fishing from my todo':
+      return 'Removed fishing from your todo';
+    case 'What is on my todo?':
+      return 'Fishing and singing in the shower';
+    case 'What day is it today?':
+      return (
+        new Date().getDate() +
+        '. of ' +
+        new Date().toLocaleString('default', { month: 'long' }) +
+        ' ' +
+        new Date().getFullYear()
+      );
```

Comment:
You don't have to create new instance each time from `new Date` you can do your `toLocalString` options one time like this
```js
  if (command === 'what day is it today?') {
    return new Date().toLocaleDateString('en-US', {
      day: 'numeric',
      year: 'numeric',
      month: 'long'
    });
  }
```